config.json

```diff
@@ -0,0 +1,163 @@
+{
+  "name": "Terraform-CLI",
+  "viewName":"Terraform CLI",
+  "type": "executer",
+  "execProgram": "node",
+  "main": "app.js",
+  "version": "1.1.3",
+  "description": "Kaholo plugin for integration with Terraform CLI.",
+  "imgUrl": "logo.png",
+  "category": ["INFRASTRUCTURE MGMT", "CI/CD"],
+  "keywords": ["cloud", "CLI", "infra", "infrastructure", "IaC", "management", "vm", "instance", "HashiCorp", "declarative"],
+  "settings": [
+    {
+      "name": "workingDir",
+      "viewName": "Default Working Directory",
+      "type": "string",
+      "description": "This is the default location where Terraform will expect to find an inventory and execution script.",
+      "placeholder": "/home/user/tmp/",
+      "learnUrl": "https://learn.hashicorp.com/tutorials/terraform/init"
+    }
+  ],
+  "methods": [
+    {
+      "name": "executeTerraformInit",
+      "viewName": "Init",
+      "params": [
+        {
+          "name": "workingDir",
```

Comment:
Please revert all the `name` fields changed in this PR to the original form. I realise they look bad, but these are the params and methods identifiers, so if we change them existing pipelines will break.

That's definitely something that needs to be fixed some day, but for now it's better work with the old incorrectly named variables, than make the clients lose their settings.

Reply:
yes